source/connection-monitoring-and-pooling/tests/pool-clear-destroy-all-conns.yml

```diff
@@ -0,0 +1,43 @@
+version: 1
+style: unit
+description: Pool MUST be able to destroy all available and in use connections
+poolOptions:
+  # ensure it's not involved by default
+  backgroundThreadIntervalMS: 10000
```

Comment:
since the immediate scheduling behavior of the background thread is optional, can we have two separate tests for it? One that sets the background thread interval relatively short (which verifies the required behavior) and one that sets it to be very long as is done here. In the test of the optional behavior, be sure to mention that it's optional in the description.

Reply:
So this test is still verifying two different behaviors, one of which is required (`closeInUseConnections: true`) and one of which is optional (immediate background thread scheduling). To allow drivers to test them separately as units, we'll need separate tests.

Here's the two tests that I'm thinking:
- "Pool clear MUST support lazily closing in use connections"
   - This would be this exact test, except `backgroundThreadInvervalMS` would be set to something low like 100ms
   - The name of the file would be `pool-clear-close-in-use.yml`
- "Pool clear SHOULD schedule the next background thread run immediately"
    - The other test can be left as-is except for the description being changed
    - The name of the file would be `pool-clear-schedule-run.yml` or something like that